mat64/triangular.go

```diff
@@ -115,7 +115,18 @@ func (t *TriDense) Dims() (r, c int) {
 }
 
 func (t *TriDense) Triangle() (n int, upper bool) {
-	return t.mat.N, t.mat.Uplo == blas.Upper
+	return t.mat.N, t.isUpper()
```

Comment:
"That case" meaning calling Triangle on a zero-sized matrix. I'd rather it return false.